<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Gestion utilisateurs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QCheckBox" name="manage_products">
     <property name="text">
      <string notr="true">Gérer les consomations</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string notr="true">Droits de l'utilsateur</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="delete_button">
     <property name="text">
      <string notr="true">Supprimer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5" colspan="2">
    <widget class="UserList" name="user_list"/>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="manage_users">
     <property name="text">
      <string notr="true">Gérer les utilisateurs</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="add_button">
     <property name="text">
      <string notr="true">Ajouter</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="manage_notes">
     <property name="text">
      <string notr="true">Gérer les notes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserList</class>
   <extends>QListWidget</extends>
   <header>gui/users_management_window</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>user_list</tabstop>
  <tabstop>manage_notes</tabstop>
  <tabstop>manage_users</tabstop>
  <tabstop>manage_products</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>delete_button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>user_list</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>Dialog</receiver>
   <slot>select_user()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manage_notes</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>on_checkbox_change(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manage_products</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>on_checkbox_change(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manage_users</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>on_checkbox_change(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_user()</slot>
  <slot>add()</slot>
  <slot>on_checkbox_change(int)</slot>
 </slots>
</ui>
